svc-bie-kafka/src/main/resources/application-prod-test.yaml

```diff
@@ -33,3 +33,13 @@ spring:
 kafka:
   topic:
     prefix: PRE
```

Comment:
It seems like this can be used to set the prefix, rather than copy-and-pasting the tweaked `kafka-topic-to-amqp-exchange-map` configuration to all yaml files.

Reply:
This one is used in KafkaConsumer class. We can auto generate this based on env variable, but it's probably a good idea to address this in a separate PR, don't want to delay the prod verification.

Reply:
How long will it take to do it correctly? We shouldn't add this tech debt if it can be implemented correctly in a short time. 

Based on this https://github.com/department-of-veterans-affairs/abd-vro/blob/8f9ce8d0cd2effa85796cc8e9a8e8ed21a071b02/svc-bie-kafka/src/main/java/gov/va/vro/services/bie/service/kafka/KafkaConsumer.java#L37-L44

It seems like this is what this prefix configuration was designed for.